toolchain/sem_ir/stringify.cpp

```diff
@@ -325,8 +326,15 @@ class Stringifier {
         sem_ir_->facet_types().Get(inst.facet_type_id);
     // Output `where` restrictions.
     bool some_where = false;
-    if (facet_type_info.other_requirements) {
-      step_stack_->PushString("...");
+    if (facet_type_info.special_requirements_mask.has(
+            SemIR::FacetTypeInfo::SpecialRequirement::Other)) {
+      step_stack_->PushString(" ...");
```

Comment:
```suggestion
      step_stack_->PushString("...");
```

Less semir churn, the lack of space isn't hurting us

Reply:
Done, though `I where .(I.V) = {} and...` seems odd to me; I'd still suggest this, or considering `TODO` as is done in formatter.